Resources/Groups/groups.yml

```diff
@@ -81,6 +81,9 @@
   - mindinfo
   - addrole
   - rmrole
+  - addobjective
```

Comment:
For merge you'd have to port this to the new system

Reply:
:(